fe/fe-core/src/main/java/com/starrocks/transaction/PublishVersionDaemon.java

```diff
@@ -568,10 +568,9 @@ private void deleteTxnLogIgnoreError(ComputeNode node, DeleteTxnLogRequest reque
             // and vacuum will clan the txn log finally if it failed.
             // make sure sync wait for the RPC call to avoid too many RPC call existed in BE/CN side
             Future<DeleteTxnLogResponse> responseFuture = lakeService.deleteTxnLog(request);
-            try {
-                DeleteTxnLogResponse response = responseFuture.get();
-            } catch (Exception e) {
-                LOG.warn("delete txn log error: " + e.getMessage());
+            DeleteTxnLogResponse response = responseFuture.get();
+            if (response.status.statusCode != 0) {
```

Comment:
should check `response` and `response.status` are not null

Reply:
updated